pkg/storaged/vgroup-details.jsx

```diff
@@ -242,10 +248,78 @@
             </Card>
         );
 
+        function is_partial_linear_lvol(block) {
+            const lvm2 = client.blocks_lvm2[block.path];
+            const lvol = lvm2 && client.lvols[lvm2.LogicalVolume];
+            return lvol && lvol.Layout == "linear" && client.lvols_status[lvol.path] == "partial";
+        }
+
+        function remove_missing() {
+            /* Calling vgroup.RemoveMissingPhysicalVolumes will
+               implicitly delete all partial, linear logical volumes.
+               Instead of allowing this, we explicitly delete these
+               volumes before calling RemoveMissingPhysicalVolumes.
+               This allows us to kill processes that keep them busy
+               and remove their fstab entries.
+
+               RemoveMissingPhysicalVolumes leaves non-linear volumes
+               alone, even if they can't be repaired anymore.  This is
+               a bit inconsistent, but *shrug*.
+            */
+
+            let usage = utils.get_active_usage(client, vgroup.path, _("delete"));
+            usage = usage.filter(u => u.block && is_partial_linear_lvol(u.block));
+
+            if (usage.Blocking) {
+                dialog_open({
+                    Title: cockpit.format(_("$0 is in use"),
+                                          vgroup.Name),
+                    Body: BlockingMessage(usage)
```

Comment:
These 5 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17226-20230918-111530-a92123e3-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)